server/server/src/middleware/mod.rs

```diff
@@ -1,12 +1,16 @@
-pub mod content_length_limit;
+use log::Level;
+
 mod central_server_only;
+pub mod content_length_limit;
 
 pub fn compress() -> actix_web::middleware::Compress {
     actix_web::middleware::Compress::default()
 }
 
 pub fn logger() -> actix_web::middleware::Logger {
-    actix_web::middleware::Logger::default()
+    actix_web::middleware::Logger::new(r#""%r" FROM: "%{Referer}i" STATUS: %s"#)
+        .log_level(Level::Debug)
+        .log_target("request_log")
```

Comment:
Supposedly this would replace:

`2025-07-09 16:57:23.039229000 [DEBUG] actix_web::middleware::logger:"POST...`

with 

`2025-07-09 16:57:23.039229000 [DEBUG] request_log:"POST...`

It doesn't do it in the in debug build, didn't build a release target, figured could check after a nightly build?